challenge.js

```diff
@@ -9,7 +9,7 @@
  */
 
 // ANSWER
-
+// By going into the console on the browser and add html element by typing 'document.createElement()' with the parameters you want.
```

Comment:
This will create a new element, but it won't add that element to the page. 1/2 credit.